examples/webpack.config.js

```diff
@@ -7,14 +7,15 @@ const path = require('path');
 const MonacoEditorSrc = path.join(__dirname, '..', 'src');
 
 module.exports = {
-  entry: {
```

Comment:
You are right!

I see that you removed the `CommonsChunkPlugin` in your merge commit so I reckon you don't need me to fix this change